graphql/util.go

```diff
@@ -114,8 +114,9 @@ func GetFormattedDate(t *time.Time, timezone string) (*string, error) {
 	return &newTime, nil
 }
 
-func getVersionBaseTasks(d data.Connector, versionID string) ([]task.Task, error) {
-	version, err := d.FindVersionById(versionID)
+func getVersionBaseTasks(versionID string) ([]task.Task, error) {
+	dc := data.DBVersionConnector{}
+	version, err := dc.FindVersionById(versionID)
```

Comment:
Rather than still using the connector, we should just call the actual model.FindVersionById function